### DISCUSSION TITLE
Monkey Backpack Counter issue

### DISCUSSION THREAD

**Discussion by GenericRsPlayer** [runelite/runelite#16521] [Ideas]
Hello! I am having an issue. I have completed 2k laps of the Ape Atoll course. I do not wish to complete another lap on the PC it to register on runelite with the command !kc Ape atoll. I am wondering if someone can make it so when you check the lap count on the monkey backpack or talks to Bobawu, that it updates the counter inside of runelite. 

there are not many people with this pack, but im sure those of us with it would be very grateful to anyone who could look into this! thank you!!
![image](https://user-images.githubusercontent.com/131407157/233487504-e9f611d3-af47-411e-a1c8-c5de58a816f4.png)

**Comment by Nightfirecat** [comment]
Related to (but not duplicate of) #13146

**Reply by GenericRsPlayer** [reply]
Thank you for tagging this
